src/test/statistics/chunk_statistics/histograms/histogram_utils_test.cpp

```diff
@@ -0,0 +1,199 @@
+  // 25 * (ipow(26, 0)) + 1
+  EXPECT_EQ(_convert_string_to_number_representation("zzzz"), 475'254ul);
+}
+
+TEST_F(HistogramUtilsTest, NumberToString) {
+  EXPECT_EQ(_convert_number_representation_to_string(0ul), "");
+
+  // 0 * (ipow(26, 3)) + 1
+  EXPECT_EQ(_convert_number_representation_to_string(1ul), "a");
+
+  // 0 * (ipow(26, 3) + ipow(26, 2) + ipow(26, 1) + ipow(26, 0)) + 1 +
+  // 0 * (ipow(26, 2) + ipow(26, 1) + ipow(26, 0)) + 1
+  EXPECT_EQ(_convert_number_representation_to_string(2ul), "aa");
+
+  // 0 * (ipow(26, 3) + ipow(26, 2) + ipow(26, 1) + ipow(26, 0)) + 1 +
+  // 0 * (ipow(26, 2) + ipow(26, 1) + ipow(26, 0)) + 1 +
+  // 0 * (ipow(26, 1) + ipow(26, 0)) + 1 +
+  // 0 * (ipow(26, 0)) + 1
+  EXPECT_EQ(_convert_number_representation_to_string(4ul), "aaaa");
+
+  // 0 * (ipow(26, 3) + ipow(26, 2) + ipow(26, 1) + ipow(26, 0)) + 1 +
+  // 0 * (ipow(26, 2) + ipow(26, 1) + ipow(26, 0)) + 1 +
+  // 0 * (ipow(26, 1) + ipow(26, 0)) + 1 +
+  // 1 * (ipow(26, 0)) + 1
+  EXPECT_EQ(_convert_number_representation_to_string(5ul), "aaab");
+
+  // 0 * (ipow(26, 3) + ipow(26, 2) + ipow(26, 1) + ipow(26, 0)) + 1 +
+  // 25 * (ipow(26, 2) + ipow(26, 1) + ipow(26, 0)) + 1 +
+  // 25 * (ipow(26, 1) + ipow(26, 0)) + 1 +
+  // 25 * (ipow(26, 0)) + 1
+  EXPECT_EQ(_convert_number_representation_to_string(18'279ul), "azzz");
+
+  // 1 * (ipow(26, 3) + ipow(26, 2) + ipow(26, 1) + ipow(26, 0)) + 1
+  EXPECT_EQ(_convert_number_representation_to_string(18'280ul), "b");
+
+  // 1 * (ipow(26, 3) + ipow(26, 2) + ipow(26, 1) + ipow(26, 0)) + 1 +
+  // 0 * (ipow(26, 2) + ipow(26, 1) + ipow(26, 0)) + 1
+  EXPECT_EQ(_convert_number_representation_to_string(18'281ul), "ba");
+
+  // 1 * (ipow(26, 3) + ipow(26, 2) + ipow(26, 1) + ipow(26, 0)) + 1 +
+  // 7 * (ipow(26, 2) + ipow(26, 1) + ipow(26, 0)) + 1 +
+  // 9 * (ipow(26, 1) + ipow(26, 0)) + 1 +
+  // 0 * (ipow(26, 0)) + 1
+  EXPECT_EQ(_convert_number_representation_to_string(23'447ul), "bhja");
+
+  // 2 * (ipow(26, 3) + ipow(26, 2) + ipow(26, 1) + ipow(26, 0)) + 1 +
+  // 3 * (ipow(26, 2) + ipow(26, 1) + ipow(26, 0)) + 1 +
+  // 4 * (ipow(26, 1) + ipow(26, 0)) + 1
+  EXPECT_EQ(_convert_number_representation_to_string(38'778ul), "cde");
+
+  // 25 * (ipow(26, 3) + ipow(26, 2) + ipow(26, 1) + ipow(26, 0)) + 1 +
+  // 25 * (ipow(26, 2) + ipow(26, 1) + ipow(26, 0)) + 1 +
+  // 25 * (ipow(26, 1) + ipow(26, 0)) + 1 +
+  // 25 * (ipow(26, 0)) + 1
+  EXPECT_EQ(_convert_number_representation_to_string(475'254ul), "zzzz");
+}
+
+TEST_F(HistogramUtilsTest, CommonPrefixLength) {
+  EXPECT_EQ(common_prefix_length("", ""), 0ul);
+  EXPECT_EQ(common_prefix_length("a", ""), 0ul);
+  EXPECT_EQ(common_prefix_length("a", "b"), 0ul);
+  EXPECT_EQ(common_prefix_length("aa", "a"), 1ul);
+  EXPECT_EQ(common_prefix_length("abcd", "abce"), 3ul);
+}
+
+/**
+ * The following tests take quite some time (multiple seconds) and are intended for exhaustive testing.
+ * TODO(tim): discuss whether to include/deactivate/remove them.
```

Comment:
@mrks @mrzzzrm Do you have any input on this?

Reply:
How about moving them to hyriseSystemTest?

Reply:
I like it, done.